zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/ZebedeeCmsService.java

```diff
@@ -141,6 +142,9 @@ public ImageService getImageService() {
     }
 
     public KafkaService getKafkaService() { return Root.zebedee.getKafkaService(); }
+
+    public StaticFilesService getStaticFilesService() { return Root.zebedee.getStaticFilesService(); }
```

Comment:
Formatting to be consistent as above if possible

Reply:
We did follow the current setup. only larger function are on multiple lines.